journald/journald.cpp

```diff
@@ -100,11 +129,22 @@ class JournaldLoggerProcess : public Process<JournaldLoggerProcess>
           return Nothing();
         }
 
-        // Write the bytes to journald.
-        Try<Nothing> result = write(readSize);
-        if (result.isError()) {
-          promise.fail("Failed to write: " + result.error());
-          return Nothing();
+        if (flags.destination_type != "sandbox") {
+          // Write the bytes to journald.
+          Try<Nothing> result = write_journald(readSize);
+          if (result.isError()) {
+            promise.fail("Failed to write: " + result.error());
+            return Nothing();
+          }
+        }
+
+        if (flags.destination_type != "journald") {
+          // Write the bytes to sandbox, with log rotation.
```

Comment:
Internal consistency, I suppose.  The matching comment above is `// Write the bytes to journald.`